.sqlfluff

```diff
@@ -0,0 +1,15 @@
+;https://docs.sqlfluff.com/en/stable/configuration.html
+;https://docs.sqlfluff.com/en/stable/rules.html#ruleref
+[sqlfluff]
+dialect=postgres
+sql_file_exts=.sql
+max_line_length=80
+
+[sqlfluff:indentation]
+allow_implicit_indents = true
+
+
+[sqlfluff:rules:capitalisation.keywords]
+capitalisation_policy = upper
+[sqlfluff:rules:capitalisation.identifiers]
+capitalisation_policy = upper
```

Comment:
Typical PostgreSQL semantics is to have identifiers lower case and keywords upper case.  I know ANSI SQL tends to upper case it all though.

Reply:
Yes, I'm old fashioned with upper case keywords. As I said this is merely an example of formatting. We'll merge when we have something that satisfies us. My guess is that we'll have to write some actual linting cases before we're done here.